build/scripts/PullRequestValidation/ValidateIssuesForPullRequest.ps1

```diff
@@ -0,0 +1,58 @@
+using module ..\GitHub\GitHubPullRequest.class.psm1
+using module ..\GitHub\GitHubIssue.class.psm1
+
+param(
+    [Parameter(Mandatory = $true)]
+    [string] $PullRequestNumber,
+    [Parameter(Mandatory = $true)]
+    [string] $Repository
+    )
+
+# Set error action
+$ErrorActionPreference = "Stop"
+
+Write-Host "Validating PR $PullRequestNumber"
+
+$pullRequest = [GitHubPullRequest]::Get($PullRequestNumber, $Repository)
+$prDescription = $pullRequest.GetBody()
+$issueRegex = "Fixes #(\d+)"
+
+if(-not $prDescription) {
+    throw "Could not find pull request description. Please make sure the pull request description contains a line that contains 'Fixes #' followed by the issue number being fixed."
+}
+
+# Get all issue matches
+$issueMatches = Select-String $issueRegex -InputObject $prDescription -AllMatches
+
+if(-not $matches) {
+    throw "Could not find issues section in the pull request description. Please make sure the pull request description contains a line that contains 'Fixes #' followed by the issue number being fixed."
```

Comment:
Could we create a comment on the pull request with this message + throw an error?

Reply:
Added.